bats/tests/preferences/verify-settings.bats

```diff
@@ -0,0 +1,50 @@
+load '../helpers/load'
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+@test 'start the app' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+proxy_set() {
+    local field=$1
+    local value=$2
+
+    payload="$(printf '{ "version": %d, "experimental": { "virtualMachine": { "proxy": { "%s": %s }}}}' "$(get_setting .version)" "$field" "$value")"
+    run rdctl api settings -X PUT --body "$payload"
+    assert_failure
+    assert_output --partial "Changing field \"experimental.virtualMachine.proxy.${field}\" via the API isn't supported"
+}
+
+@test 'complain about windows-specific vm settings' {
+    skip_on_windows
+    run rdctl api /settings
+    assert_success
+    run jq .experimental.virtualMachine.proxy.enabled <<<"$output"
```

Comment:
Hmm, I added a comment here before, but Github seems to have swallowed it...

Use `jq_output` here; it is tidier and will throw an error if the field does not exist whereas plain `jq` will just return `null`:

```console
$ echo {} | jq .experimental.virtualMachine.proxy.enabled; echo $?
null
0
```

```suggestion
    run jq_output .experimental.virtualMachine.proxy.enabled
```